### ISSUE TITLE
scale/interpretation of Aspect values in HK Report

### ISSUE THREAD

**Issue by nicHoch** [i4Ds/STIXCore#281]
the diodes values in HK 2 report: 

HK_ASP_PHOTOA0_V, HK_ASP_PHOTOA1_V, HK_ASP_PHOTOB0_V, HK_ASP_PHOTOB1_V
NIX00078 - NIX00081

are aggregated over 64s (configurable) each 4s (hard coded) as a uint32
the raw values are uint16 coming from the hardware.

```
static void timerCallback(rtems_id _timerId, void* _arg)
{
	FSW_UNUSED(_timerId);
	FSW_UNUSED(_arg);

	uint16_t aspDatum[4];
	Aspect_getQLData(aspDatum);
	aspectHKsums[0] += aspDatum[0];
	aspectHKsums[1] += aspDatum[1];
	aspectHKsums[2] += aspDatum[2];
	aspectHKsums[3] += aspDatum[3];

	rtems_timer_reset(timerId);
}
```

then at HK report creation the accumulated values where readout and reset:

```
void AspectCtrl_getAndResetHK(uint32_t* aspA0, uint32_t* aspA1, uint32_t* aspB0, uint32_t* aspB1)
{
	ASSERT(aspA0 && aspA1 && aspB0 && aspB1);

	// The algorithm writeup foresees that the 4 LSBs are dropped to make the aspect values fit. This is done under the assumption that
	// the HK interval is 64 seconds.
	*aspA0 = aspectHKsums[0] >> 4;
	*aspA1 = aspectHKsums[1] >> 4;
	*aspB0 = aspectHKsums[2] >> 4;
	*aspB1 = aspectHKsums[3] >> 4;
	
	aspectHKsums[0] = 0;
	aspectHKsums[1] = 0;
	aspectHKsums[2] = 0;
	aspectHKsums[3] = 0;
}
```

so from the uint32 we truncate the 4 LSB

while getting the values:

and we write 16bits into the tm stream

```
fsw_ret_e TMForm_createTM_3_25_SID_2(const HKData_params_sid2_s* in_HKparams)
{
...
u16_to_pu8(&rawDataBuffer[TMData_DATA_FIELD + dataFieldIndex], in_HKparams->aspDiodeAccumulatorA0); dataFieldIndex += 2;
	u16_to_pu8(&rawDataBuffer[TMData_DATA_FIELD + dataFieldIndex], in_HKparams->aspDiodeAccumulatorA1); dataFieldIndex += 2;
	u16_to_pu8(&rawDataBuffer[TMData_DATA_FIELD + dataFieldIndex], in_HKparams->aspDiodeAccumulatorB0); dataFieldIndex += 2;
	u16_to_pu8(&rawDataBuffer[TMData_DATA_FIELD + dataFieldIndex], in_HKparams->aspDiodeAccumulatorB1); dataFieldIndex += 2;
...
```

so basically we sending down bit 4-20 from the accumulated values in the TM.
the time stamp in the HK report header is the current time when crating the report. so the aggregate aspect values are from the past 64 seconds.  

the processing pipeline does the following transformations:

LB: nothing
L0: read in 16bit for each diode value NIX00078 - NIX00081
L1: raw to engineering conversion with:  

`3.276 / 4095 * raw`  and attach the unit `V` 

L2/Aux the Voltage from L1 is send to IDL for the pointing calculations

so in the code i do not see any un-shifting but this could somehow be included in the transformation formula from raw to V?

**Comment by nicHoch** [comment]
discussed on colocation and there is nothing to do
